roadmap-site/README.md

```diff
@@ -0,0 +1,109 @@
+# Challenges
+
+## User Object
+
+{
+"user_id":<memberid>,
+"first_name":string,
+"last_name":string,
+"yoe": number,
+"company": string,,
+"designation": string,
+"img": string,
+"github_id": string,
+"linkedin_id": string,
+"twitter_id": string,
+"instagram_id": string,
+}
```

Comment:
`isMember` seems to be missing. Is it intentional or accidental?

Reply:
Accidental, I have added this